prow/jobs/jinja/config/label_sync.jinja2

```diff
@@ -322,6 +337,11 @@ default:
       target: prs
       prowPlugin: require-matching-label
       addedBy: prow
+    - color: e11d21
+      description: Indicates an issue needs some investigation.
+      name: needs-investigation
```

Comment:
`priority/needs-investigation`?

Reply:
We have the same `priority/*` labels as the ones defined in kubernetes. I think we should keep it that way + we already have `priority/awaiting-more-evidence` which doesn't necessarily mean that we *need* to investigate an issue. Thoughts?